plugins/inputs/prometheus/prometheus.go

```diff
@@ -487,10 +492,9 @@ func (p *Prometheus) Stop() {
 func init() {
 	inputs.Add("prometheus", func() telegraf.Input {
 		return &Prometheus{
-			ResponseTimeout: config.Duration(time.Second * 3),
```

Comment:
Do you set the default somewhere else? To me it seems the new default for `timeout` is **zero** but the config says `5s` (which didn't even match before :-S)...

Reply:
That's the default for [HTTPClientConfig](https://github.com/influxdata/telegraf/blob/aade2e4c4150cd3b68f4dde2a2e04ad63ae93cfd/plugins/common/http/config.go#L51-L54)

Reply:
Thanks for the clarification!